### ISSUE TITLE
Disable logging

### ISSUE THREAD

**Issue by Pilli** [akhikhl/gretty#140]
Hi.
Is possible disable gretty logging?
I have old application code that directly references log4j appender. When i run with gretty this Exception is raised:
```
.......
Caused by: org.apache.commons.discovery.DiscoveryException: Class org.apache.commons.logging.impl.SLF4JLogFactory does not implement org.apache.commons.logging.LogFactory
.......
```
I can't use [log4j-over-slf4j](http://www.slf4j.org/legacy.html#losFail).
If not is possible would be nice have a option to do this. Something like this:
```
gretty {
logging = 'disable'
}
```
or
```
gretty {
enableLoggin = false // true by default
}
```

Thanks and best regards.

**Comment by akhikhl** [comment]
Hi Philli,
The problem is fixed in Gretty 1.1.9-SNAPSHOT. Now Gretty isolates it's own (logback-based) logging from the web-application. It does it on the classloader level, so web-application will not see any logging libraries or configurations from Gretty.
Gretty release 1.1.9 is due tomorrow, 03.03.2015, please feel free trying it out.

**Comment by Pilli** [comment]
Hi Andrey.
It works fine now.

Thanks and best regards.
